### PR TITLE
Fixes and changes

### PR DESCRIPTION
**Pull request by weigo87**
- added sounds from ogg2 (only if they are missing)
- changed AR4230(N).mos files for EE and non-EE version (EE has larger images)
- changed tiz format to tbc format
- added a7_tools.tpa by Argent77 for HANDLE_TILECONV function
- added in %MOD_FOLDER%/Tools/Tileconv all tileconv versions

### CODE REVIEW

File: SoS/SoS.tp2 [hunk 1]
```diff
@@ -1082,15 +1090,21 @@ BEGIN @21011
                 ~%MOD_FOLDER%/Pvrz/Alt/A4230N17.PVRZ~ ~override~
                 ~%MOD_FOLDER%/Pvrz/Alt/A4230N18.PVRZ~ ~override~
                 ~%MOD_FOLDER%/Pvrz/Alt/A4230N19.PVRZ~ ~override~
+                ~%MOD_FOLDER%/Mos/ALT/ee/AR4230.MOS~ ~override~
+                ~%MOD_FOLDER%/Mos/ALT/ee/AR4230N.MOS~ ~override~
     END
     ELSE BEGIN
-        COPY_LARGE    ~%MOD_FOLDER%/Tiz/ALT/AR4230.TIS~ ~override~
-                    ~%MOD_FOLDER%/Tiz/ALT/AR4230N.TIS~ ~override~
+        INCLUDE ~%MOD_FOLDER%/lib/a7_tools.tpa~
```
Comment (Roberciiik) [inline SoS/SoS.tp2:1097]: It seems that this file is already included in line 1016

Comment (ALIENQuake) [reply SoS/SoS.tp2:1097]: @Roberciiik Yes but it's for the previous component.

Comment (Roberciiik) [reply SoS/SoS.tp2:1097]: It could be added to ALWAYS phase then. But this is fine too.

**Comment by ALIENQuake** [comment]
@weigo87 A few minor things:
- can you update `a7_tools.tpa` to version 1.1 https://github.com/Argent77/DjinniCompanion/blob/fefc768d00fc70c14db19707191d077907be0a35/DjinniCompanion/lib/a7_tools.tpa
- since you added "Tools\tileconv\Win32\**x86\tileconv.exe**" and "Tools\tileconv\Win32\**x86_64\tileconv.exe**" can you remove  "Tools\tileconv\**Win32\tileconv.exe**" since it's not necessary

**Comment by Roberciiik** [comment]
I recommend to use a7_tools version 1.2, you can use the one from BP-BGT-Worldmap: https://github.com/SpellholdStudios/BP-BGT-Worldmap/commit/e51320424bdb64dc4726f35c47dab783daf09507#diff-e597496beb9a63b27027ff1a9948130e433d32adbcf7cbe30ca0ddc174571ce7

**Comment by weigo87** [comment]
both done

**Comment by ALIENQuake** [comment]
@weigo87 @Roberciiik Argent77 released an updated package https://github.com/InfinityTools/tileconv/releases so I also updated it here.

Thank you for your contribution!

**Pull request merged**